src/ext/java/org/opentripplanner/ext/gtfsgraphqlapi/datafetchers/TripOccupancyImpl.java

```diff
@@ -0,0 +1,31 @@
+package org.opentripplanner.ext.gtfsgraphqlapi.datafetchers;
+
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import org.opentripplanner.ext.gtfsgraphqlapi.generated.GraphQLDataFetchers;
+import org.opentripplanner.ext.gtfsgraphqlapi.model.TripOccupancy;
+
+public class TripOccupancyImpl implements GraphQLDataFetchers.GraphQLTripOccupancy {
+
+  @Override
+  public DataFetcher<String> occupancyStatus() {
+    return env -> {
+      var occupancyStatus = getSource(env).occupancyStatus();
+      return switch (occupancyStatus) {
+        case NO_DATA_AVAILABLE -> "NO_DATA_AVAILABLE";
```

Comment:
As above, please map into the proper schema type. It helps to avoid problems like #5368